src/main/com/smithsmodding/armory/common/logic/initialization/CommonSystemInitializer.java

```diff
@@ -147,7 +83,9 @@ private static void tryRemoveRecipeFromGame(@Nonnull IRecipe recipe, @Nonnull It
                         try {
                             iterator.remove();
                             return;
-                        } catch (IllegalStateException ex) {
+                        }
+                        catch (Exception ex)
+                        {
                             ModLogger.getInstance().info("Could not remove recipe of: " + ItemStackHelper.toString(recipe.getRecipeOutput()));
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Define a constant instead of duplicating this literal "Could not remove recipe of: " 3 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1192)